src/App.tsx

```diff
@@ -1,203 +1,64 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
-
+import React, { useContext, useEffect, useState } from 'react';
+import cn from 'classnames';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorContext, ErrorsMessageContext } from './components/ErrorsContext';
+import { ShowErrorsMessage } from './utils/fetchClient';
+
+const USER_ID = 11969;
+
+// 11969
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const { isError, setIsError } = useContext(ErrorContext);
+  const { errorsMesage, setErrorsMesage } = useContext(ErrorsMessageContext);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((data) => {
+        setTodos(data);
+      })
+      .catch(() => {
+        setIsError(false);
+        setErrorsMesage('load');
+      });
+  }, [setErrorsMesage, setIsError]);
```

Comment:
no needs to add these dependencies 
```suggestion
  }, []);
```